### PR TITLE
BrainIO Spike Times Updates

### PR DESCRIPTION
**Pull request by jjpr-mit**
https://github.com/brain-score/brainio/pull/17
BrainIO has been modified to support processing spike times.  Updating Brain-Score for compatibility.

### CODE REVIEW

File: tests/test_benchmarks/test___init__.py [hunk 1]
```diff
@@ -423,7 +423,7 @@ class TestVisualDegrees:
         pytest.param('tolias.Cadena2017-pls', 6, '0fe27ddd5b9ea701e380063dc09b91234eba3551', approx(.29641, abs=.0001),
                      marks=[pytest.mark.private_access]),
     ])
-    def test_amount_gray(self, benchmark, candidate_degrees, image_id, expected):
+    def test_amount_gray(self, benchmark, candidate_degrees, image_id, expected, brainio_home, resultcaching_home, brainscore_home):
```
Comment (mschrimpf) [inline tests/test_benchmarks/test___init__.py:426]: I'm not the biggest fan of these configuration parameters in the method, is there a way to not have them here?

Comment (jjpr-mit) [reply tests/test_benchmarks/test___init__.py:426]: They're not really parameters, they're fixtures.  Without them the test leaves files behind.  It's a problem when testing locally.  I could [convert](https://docs.pytest.org/en/latest/how-to/fixtures.html#autouse-fixtures-fixtures-you-don-t-have-to-request) those fixtures to `autouse=True` I suppose.

File: tests/test_benchmarks/test___init__.py [hunk 2]
```diff
@@ -230,12 +230,12 @@ def test_MajajHong2015(self, benchmark, expected):
     def run_test(self, benchmark, file, expected):
         benchmark = benchmark_pool[benchmark]
         precomputed_features = Path(__file__).parent / file
-        precomputed_features = BehavioralAssembly(xr.load_dataarray(precomputed_features))
+        precomputed_features = BehavioralAssembly.from_files(precomputed_features, stimulus_set_identifier=benchmark._assembly.stimulus_set.identifier, stimulus_set=benchmark._assembly.stimulus_set)
```
[the remark this answers is not in the document]
Comment (jjpr-mit) [reply tests/test_benchmarks/test___init__.py:233]: We could make `merge_stimulus_set_meta` conditional, skip it if `self.stimulus_set` is `None`.  Assembly classes using `StimulusReferenceAssemblyLoader` can already deal with `None` in the `attrs`, we could extend that to the `StimulusMergeAssemblyLoader` loader.

File: brainscore/benchmarks/imagenet_c.py [hunk 3]
```diff
@@ -186,14 +188,15 @@ class SampledStimulusSetLoader(StimulusSetLoader):
...
     def load(self):
         stimulus_set = pd.read_csv(self.csv_path)[::self.sampling_factor]
-        stimulus_set = StimulusSet(stimulus_set)
-        stimulus_set.image_paths = {row['image_id']: os.path.join(self.stimuli_directory, row['filename'])
-                                    for _, row in stimulus_set.iterrows()}
-        assert all(os.path.isfile(image_path) for image_path in stimulus_set.image_paths.values())
+        self.correct_stimulus_id_name(stimulus_set)
+        stimulus_set = self.stimulus_set_class(stimulus_set)
+        stimulus_set.stimulus_paths = {row['stimulus_id']: Path(self.stimuli_directory) / row['filename']
+                                       for _, row in stimulus_set.iterrows()}
```
[the remark this answers is not in the document]
Comment (jjpr-mit) [reply brainscore/benchmarks/imagenet_c.py:201]: Added comment in [dd6f031](https://github.com/brain-score/brain-score/pull/326/commits/dd6f031c623b42be002a1dfc2f0459d247e9e010).

**Pull request merged**